tests/utils/admincli/vmgroupmgmt.go

```diff
@@ -52,3 +52,15 @@ func ReplaceVmFromVmgroup(ip, name, vmName string) ([]byte, error) {
 	log.Printf("Replacing VM [%s] from a vmgroup [%s] on esx [%s] \n", vmName, name, ip)
 	return ssh.InvokeCommand(ip, admincli.ReplaceVmFrmVmgroup+"--name="+name+" --vm-list="+vmName)
 }
+
+// Initialize the (local) Single Node Config DB
+func ConfigInit(ip string) ([]byte, error) {
```

Comment:
We should improve this function to support both SingleNode and MultiNode. But I'm fine to take care of this whenever we need it afterwards.